sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -185,6 +190,8 @@ class APISpec extends Specification {
         premiumBlobServiceClient = setClient(premiumCredential)
         managedDiskServiceClient = setClient(managedDiskCredential)
         versionedBlobServiceClient = setClient(versionedCredential)
+        dataLakeBlobServiceClient = setClient(dataLakeCredential)
+        dataLakeBlobServiceAsyncClient = getServiceAsyncClient(dataLakeCredential)
```

Comment:
Shouldn't this go in the datalake package if it reuires ADLS?

Reply:
At this point we have this API on both blob and datalake file clients. Tests for blob client should live close to it. However they require HNS enabled that's why this came here. BlobClient is tested by BlobTest that requires datalake account.

Whether this API is going to stay at blobclient is an open question now (this feature isn't going public yet).